lua/nvim-tree/view.lua

```diff
@@ -335,8 +335,10 @@ end
 
 function M.focus(winnr, open_if_closed)
   local wnr = winnr or M.get_winnr()
-
-  if a.nvim_win_get_tabpage(wnr or 0) ~= a.nvim_win_get_tabpage(0) then
+  if not a.nvim_win_is_valid(wnr) then
```

Comment:
I don't think we need this any more.

I tested successfully without these changes.